### ISSUE TITLE
Error when Selecting Fig from Variable Explorer

### ISSUE THREAD

**Issue by PowerPlum** [spyder-ide/spyder#5213]
## Description

**What steps will reproduce the problem?**

1. 
2. 
3. 

**What is the expected output? What do you see instead?**

**Please provide any additional information below**

  File "c:\python36-32\lib\site-packages\spyder\widgets\variableexplorer\collectionseditor.py", line 519, in createEditor
    readonly=readonly)
  File "c:\python36-32\lib\site-packages\spyder\widgets\variableexplorer\collectionseditor.py", line 1308, in setup
    self.data_copy = copy.deepcopy(data)
  File "c:\python36-32\lib\copy.py", line 180, in deepcopy
    y = _reconstruct(x, memo, *rv)
  File "c:\python36-32\lib\copy.py", line 280, in _reconstruct
    state = deepcopy(state, memo)
  File "c:\python36-32\lib\copy.py", line 150, in deepcopy
    y = copier(x, memo)
  File "c:\python36-32\lib\copy.py", line 240, in _deepcopy_dict
    y[deepcopy(key, memo)] = deepcopy(value, memo)
  File "c:\python36-32\lib\copy.py", line 161, in deepcopy
    y = copier(memo)
  File "c:\python36-32\lib\site-packages\matplotlib\transforms.py", line 125, in __copy__
    "Consider using frozen() instead.")
NotImplementedError: TransformNode instances can not be copied. Consider using frozen() instead.

## Version and main components

* Spyder Version: 3.2.3 
* Python Version: 3.6.2
* Qt Versions: 5.9.1, PyQt5 5.9 on Windows

## Dependencies
```
pyflakes

**Comment by ccordoba12** [comment]
What do you mean by selecting Fig? Please be more specific.

**Comment by PowerPlum** [comment]
Hi,
Sorry - this was an automated error code generated by Spyder, and I presed OK before reading it properly. 
The error occured when creating a plot using Matplotlib within Spyder. The code was as follows:
```
import matplotlib.pyplot as plt
fig = plt.figure(figsize=(20, 10))
```
A variable appeared in Variable Explorer called fig. I double clicked on it to examine, and this caused an error. 
I've added a screen shot to this note. 
![spyder and matplotlib error](https://user-images.githubusercontent.com/29338325/30364636-59a4c104-9865-11e7-96d0-ac35e8eedd5b.png)
Let me know if there is anything else I can provide
Thanks
